optimism2/dex/insert_wardenswap.sql

```diff
@@ -0,0 +1,135 @@
+CREATE OR REPLACE FUNCTION dex.insert_wardenswap(start_ts timestamptz, end_ts timestamptz=now()) RETURNS integer
+LANGUAGE plpgsql AS $function$
+DECLARE r integer;
+BEGIN
+WITH rows AS (
+    INSERT INTO dex.trades (
+        block_time,
+        token_a_symbol,
+        token_b_symbol,
+        token_a_amount,
+        token_b_amount,
+        project,
+        version,
+        category,
+        trader_a,
+        trader_b,
+        token_a_amount_raw,
+        token_b_amount_raw,
+        usd_amount,
+        token_a_address,
+        token_b_address,
+        exchange_contract_address,
+        tx_hash,
+        tx_from,
+        tx_to,
+        trace_address,
+        evt_index,
+        trade_id
+    )
+    SELECT
+        dexs.block_time,
+        erc20a.symbol AS token_a_symbol,
```

Comment:
ah good catch. The ETH placeholder I've been using for Optimism is `0xdeaddeaddeaddeaddeaddeaddeaddeaddead0000` since that matches what deposits come in as. I think we should force it to this, I'll build that in.